projects/plutus/inc/current_sense.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+// Current sense module for the LTC2484 ADC
+
+#include "ltc_adc.h"
+#include "status.h"
+
+typedef void (*CurrentSenseCallback)(int32_t current, void *context);
+
+typedef struct {
+  int32_t voltage;
+  int32_t current;
+} CurrentSenseValue;
+
+// User-defined points for two-point calibration
+typedef struct {
+  CurrentSenseValue zero_point;
+  CurrentSenseValue max_point;
+} CurrentSenseCalibrationData;
+
+typedef struct {
+  LtcAdcStorage adc_storage;
+  CurrentSenseCalibrationData data;
+  CurrentSenseValue value;
+  int32_t offset;
+  bool offset_flag;
```

Comment:
Can you use a more descriptive name that makes it more obvious that this is a pending check?

Reply:
Fixed